releases/release-1.27/release-notes/maps/pr-116043-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 116043
+releasenote:
+  text: ' From now on, the HPA controller will return an error for the container resource
```

Comment:
Remove leading space after `'`. Seems like a few other files in the PR has the same. Please clean them all up ?

Reply:
sure